### DISCUSSION TITLE
Disable automated upgrades

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#8471] [General]
I've installed codeql with homebrew and I want to manage codeql version with homebrew instead. So how can I disable auto updates?

![111](https://user-images.githubusercontent.com/1357701/158706504-f2d2408a-16da-4a2a-ad14-7e3f024f5523.jpg)

**Comment by hvitved** [comment]
You can set the option `Code QL › Cli: Executable Path` to point to the homebrew binary. However, if that binary is already on the `PATH`, then the VS Code extension should pick it up automatically (you will have to close and reopen VS Code after adding it to `PATH`).

**Reply by CaledoniaProject** [reply]
So there's no automatic updates. vscode is installing codeql because it's not in $PATH?

**Reply by hvitved** [reply]
Correct. But I think it will automatically update automatically downloaded CLIs when they become out-of-date.

**Reply by CaledoniaProject** [reply]
Can I disable this feature? Because the vscode starter project need to be updated as well.
